src/modules/apps/exec.ts

```diff
@@ -0,0 +1,53 @@
+import * as WebSocket from 'ws'
+import * as url from 'url'
+import { getAccount, getToken, getWorkspace } from '../../conf'
+import { getManifest } from '../../manifest'
+import { region } from '../../env'
+import { toMajorRange } from '../../locator'
+const EOT = '\x04'
+export default async (command: string, options) => {
+  const manifest = await getManifest()
+  const { name, vendor, version, builders } = manifest
+  const { dotnet, node, 'service-js': serviceJs } = builders
+  if (!dotnet && !node && !serviceJs) {
+    return
+  }
+  const majorRange = toMajorRange(version)
+  const host = `${name}.${vendor}.${region()}.vtex.io`
+  const path = `/${getAccount()}/${getWorkspace()}/_exec`
+  const clientOptions = {
+    headers: {
+      Authorization: getToken(),
+      Host: host,
+      'X-Vtex-Runtime-Api': 'true',
```

Comment:
I think there's another header that should be added, which specifies the cluster if it's defined:
https://github.com/vtex/toolbelt/blob/3f4941d2edebc2e493367fbea75aae851a2c8c1a/src/clients/index.ts#L30-L37